src/test/scala/chiselTests/ProbeSpec.scala

```diff
@@ -0,0 +1,144 @@
+package chiselTests
+
+import chisel3._
+import circt.stage.ChiselStage
+
+class ProbeSpec extends ChiselFlatSpec with Utils {
+  // Strip SourceInfos and split into lines
+  private def processChirrtl(chirrtl: String): Array[String] =
+    chirrtl.split('\n').map(line => line.takeWhile(_ != '@').trim())
+
+  "Simple probe usage" should "work" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(
+      new Module {
+        val a = IO(Output(Probe(Bool())))
+
+        val w = WireInit(Bool(), false.B)
+        val w_probe = Probe.probe(w)
+        Probe.define(a, w_probe)
```

Comment:
I think we need an explanation.md to explain the different pieces here and how they are subtly different. My internal model is:

line 14 creates an "output probe port -- call it a". Just like any IO port you need to drive it with something.

Line 18 is the thing that drives it, roughly equivalent to 

a := w_probe

line 17 does something -- starts a probe and ties it to w? Why do we need this step vs just saying line 18
Probe.define(a, w)?

Is my intuition something like existing chisel ops:

a := w_probe.asProbe

Missing from this example: whats the right operation to forward through hierarchies? To connect siblings?

then finally there is the "end user" .read, force operations.